quickstarts/microsoft.kubernetes/aks-helm/nested_template/helm.bicep

```diff
@@ -0,0 +1,95 @@
+@description('Location of all resources to be deployed')
+param location string = resourceGroup().location
+
+@description('The base URI where artifacts required by this template are located')
+param _artifactsLocation string = deployment().properties.templateLink.uri
+
+@description('The sasToken required to access artifacts')
+@secure()
+param _artifactsLocationSasToken string = ''
+
+param clusterName string = ''
+
+param utcValue string = utcNow()
+
+@description('Public Helm Repo Name')
+param helmRepo string = 'azure-marketplace'
+
+@description('Public Helm Repo URL')
+param helmRepoURL string = 'https://marketplace.azurecr.io/helm/v1/repo'
+
+@description('Public Helm App')
+param helmApp string = 'azure-marketplace/wordpress'
+
+@description('Public Helm App Name')
+param helmAppName string = 'my-wordpress'
+
+var installScriptUri = uri(_artifactsLocation, 'scripts/helm.sh${_artifactsLocationSasToken}')
+
+var identityName = 'scratch${uniqueString(resourceGroup().id)}'
+var roleDefinitionId = resourceId('Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')
+var roleAssignmentName = guid(identityName, roleDefinitionId)
```

Comment:
this needs to contain the principal, role and scope as the seed - the identity name should be the resourceId of the MSI in the seed, e.g.

```bicep
guid(roleDefinitionId, managedIdentity.id, resourceGroup().id)
```